content/en/docs/reference/glossary/cloud-controller-manager.md

```diff
@@ -4,16 +4,16 @@ id: cloud-controller-manager
 date: 2018-04-12
 full_link: /docs/tasks/administer-cluster/running-cloud-controller/
 short_description: >
-  Cloud Controller Manager is an alpha feature in 1.8. In upcoming releases it will be the preferred way to integrate Kubernetes with any cloud.
+  Cloud Controller Manager is a Kubernetes component that embeds cloud-specific control logic.
 
 aka: 
 tags:
 - core-object
 - architecture
 - operation
 ---
- Cloud Controller Manager is an alpha feature in 1.8. In upcoming releases it will be the preferred way to integrate Kubernetes with any cloud.
+ Cloud Controller Manager is a Kubernetes component that embeds cloud-specific control logic.
 
 <!--more--> 
 
-Kubernetes v1.6 contains a new binary called cloud-controller-manager. cloud-controller-manager is a daemon that embeds cloud-specific control loops. These cloud-specific control loops were originally in the kube-controller-manager. Since cloud providers develop and release at a different pace compared to the Kubernetes project, abstracting the provider-specific code to the cloud-controller-manager binary allows cloud vendors to evolve independently from the core Kubernetes code.
+Originally part of the kube-controller-manager, these cloud specific control logic abstract the provider code to the Cloud Controller Manager and enable cloud providers to develop and release at a different pace compared to the Kubernetes project.
```

Comment:
@sftim, This is quite different from your edits in pull request #19705.
Line 19 is hard to parse.

Reply:
@davivcgarcia - I'd be happy to collaborate on getting the best bits of this PR and #19705 

(it's been a while since I sent #19705 in, and as far as I know that PR was good to merge, but it wasn't for me to review).

Reply:
@kbhawkey : what about `Originally part of the kube-controller-manager, the cloud-controller-manager is responsible to decoupling the interoperability between Kubernetes and the underlying cloud infrastructure, enabling cloud providers to release features at a different pace compared to the main project.` ?